tests/terraform/checks/resource/aws/test_CloudFrontResponseHeaderStrictTransportSecurity.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from pathlib import Path
+
+from checkov.terraform.runner import Runner
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.checks.resource.aws.CloudFrontResponseHeaderStrictTransportSecurity import check
+
+
+class TestLambdaEnvironmentCredentials(unittest.TestCase):
```

Comment:
The class name is leftover from the copy/paste

Reply:
😄 nice catch, yeah totally copy paste left over